components/shooter.py

```diff
@@ -132,4 +142,5 @@ def execute(self) -> None:
         self.inclinator.set(inclinator_speed)
 
         flywheel_request = VelocityVoltage(self.desired_flywheel_speed)
-        self.flywheel.set_control(flywheel_request)
+        self.flywheel_left.set_control(flywheel_request)
+        self.flywheel_right.set_control(Follower(TalonIds.shooter_flywheel_left, True))
```

Comment:
A follower command should only be set once at robot start, like CTRE's example: https://github.com/CrossTheRoadElec/Phoenix6-Examples/blob/536bfad58a4c94f0e3b766aff71d763322b35a0b/python/ArcadeDrive/robot.py#L32